speakers/speakers.jsonp

```diff
@@ -201,5 +216,5 @@ process_speakers([
         "F# in the real world", 
         "Taming cloud complexity with F# DSLs" ],
     "tags": [ "dsl", "cloud", "distributed systems" ]
-}
+},
```

Comment:
You added an extra comma here - can you remove it?  It will cause a syntax error in parsing.